.organizationmap

```diff
@@ -561,51 +637,68 @@ Red Hat <contact@redhat.com> Huamin Chen <hchen@redhat.com>
+Red Hat <contact@redhat.com> Justin Caratzas <jcaratza@redhat.com>
 Red Hat <contact@redhat.com> Kaleb S. Keithley <kkeithle@redhat.com>
-Red Hat <contact@redhat.com> Kamoltat (Junior) Sirivadhna <ksirivad@redhat.com>
+Red Hat <contact@redhat.com> Kalpesh Pandya <kapandya@redhat.com>
+Red Hat <contact@redhat.com> Kamoltat Sirivadhna <ksirivad@redhat.com>
 Red Hat <contact@redhat.com> Kanika Murarka <kmurarka@redhat.com>
 Red Hat <contact@redhat.com> Karun Josy <kjosy@redhat.com>
 Red Hat <contact@redhat.com> Kefu Chai <kchai@redhat.com>
 Red Hat <contact@redhat.com> Ken Dreyer <kdreyer@redhat.com>
-Red Hat <contact@redhat.com> Kotresh HR <khiremat@redhat.com>
+Red Hat <contact@redhat.com> Kotresh Hiremath Ravishankar <khiremat@redhat.com>
 Red Hat <contact@redhat.com> Laura Flores <lflores@redhat.com>
 Red Hat <contact@redhat.com> Loic Dachary <ldachary@redhat.com>
 Red Hat <contact@redhat.com> Loic Dachary <loic-201408@dachary.org>
 Red Hat <contact@redhat.com> Luis Pabón <lpabon@redhat.com>
 Red Hat <contact@redhat.com> Marcus Watts <mwatts@redhat.com>
 Red Hat <contact@redhat.com> Mark Kogan <mkogan@redhat.com>
 Red Hat <contact@redhat.com> Mark Nelson <mnelson@redhat.com>
+Red Hat <contact@redhat.com> Mary Frances <mhull@redhat.com>
 Red Hat <contact@redhat.com> Matan Breizman <mbreizma@redhat.com>
 Red Hat <contact@redhat.com> Matt Benjamin <mbenjamin@redhat.com>
 Red Hat <contact@redhat.com> Mehdi Abaakouk <sileht@redhat.com>
+Red Hat <contact@redhat.com> Melissa Li <melissali@redhat.com>
+Red Hat <contact@redhat.com> Michael J. Kidd <linuxkidd@redhat.com>
+Red Hat <contact@redhat.com> Michaela Lang <milang@redhat.com>
 Red Hat <contact@redhat.com> Mike Christie <mchristi@redhat.com>
 Red Hat <contact@redhat.com> Mike Hackett <mhackett@redhat.com>
 Red Hat <contact@redhat.com> Mike Perez <miperez@redhat.com>
 Red Hat <contact@redhat.com> Milan Broz <mbroz@redhat.com>
 Red Hat <contact@redhat.com> Milind Changire <mchangir@redhat.com>
 Red Hat <contact@redhat.com> Nathan Weinberg <nweinber@redhat.com>
+Red Hat <contact@redhat.com> Neeraj Pratap Singh <neesingh@redhat.com>
 Red Hat <contact@redhat.com> Neha Ojha <nojha@redhat.com>
 Red Hat <contact@redhat.com> Neil Levine <nlevine@redhat.com>
+Red Hat <contact@redhat.com> Nikhilkumar Shelke <nshelke@redhat.com>
 Red Hat <contact@redhat.com> Nilamdyuti Goswami <ngoswami@redhat.com>
+Red Hat <contact@redhat.com> Nithya Balachandran <nibalach@redhat.com>
 Red Hat <contact@redhat.com> Nitzan Mordechai <nmordech@redhat.com>
 Red Hat <contact@redhat.com> Nizamudeen A <nia@redhat.com>
 Red Hat <contact@redhat.com> Noah Watkins <nwatkins@redhat.com>
+Red Hat <contact@redhat.com> Omri Zeneva <ozeneva@redhat.com>
+Red Hat <contact@redhat.com> Ondrej Mosnacek <omosnace@redhat.com>
 Red Hat <contact@redhat.com> Or Friedmann <ofriedma@redhat.com>
 Red Hat <contact@redhat.com> Orit Wasserman <owasserm@redhat.com>
+Red Hat <contact@redhat.com> parth-gr <paarora@redhat.com>
```

Comment:
Updated to use my actual name instead of github profile name
`Red Hat <contact@redhat.com> Parth Arora <paarora@redhat.com>`

Reply:
thanks, i updated the name here and added a mapping from parth-gr -> Parth Arora

Reply:
Thanks